core-api/src/main/java/com/optimizely/ab/internal/ExperimentUtils.java

```diff
@@ -77,4 +85,15 @@ public static boolean isUserInExperiment(@Nonnull ProjectConfig projectConfig,
 
         return false;
     }
+
+    public static Boolean resolveAudienceConditions(@Nonnull ProjectConfig projectConfig,
+                                            @Nonnull Experiment experiment,
+                                            @Nonnull Map<String, ?> attributes) {
+
+        ConditionUtils.resolveAudienceIdConditions(projectConfig, experiment.getAudienceConditions());
```

Comment:
As mentioned above, it's not clear to me what we accomplish by calling `ConditionUtils.resolveAudienceIdConditions`.